packages/utils/modify-config-helper.ts

```diff
@@ -44,59 +60,59 @@ module.exports = function modifyHelperUtil(action, generator, configFile, packag
 					" not found. Please specify a valid path to your webpack config like " +
 					chalk.white("$ ") +
 					chalk.cyan(`webpack-cli ${action} webpack.dev.js`) +
-					"\n"
+					"\n",
 			);
 			return;
 		}
 	}
 
-	const env = yeoman.createEnv("webpack", null);
-	const generatorName = `webpack-${action}-generator`;
+	const env: any = yeoman.createEnv("webpack", null);
+	const generatorName: string = `webpack-${action}-generator`;
 
 	if (!generator) {
 		generator = class extends Generator {
-			initializing() {
-				packages.forEach(pkgPath => {
-					return this.composeWith(require.resolve(pkgPath));
+			public initializing() {
+				packages.forEach((pkgPath: string) => {
+					return (this as any).composeWith(require.resolve(pkgPath));
 				});
 			}
 		};
 	}
 	env.registerStub(generator, generatorName);
 
 	env.run(generatorName).on("end", () => {
-		let configModule;
+		let configModule: object;
 		try {
-			const configPath = path.resolve(process.cwd(), ".yo-rc.json");
-			configModule = require(configPath);
+			const confPath: string = path.resolve(process.cwd(), ".yo-rc.json");
+			configModule = require(confPath);
 			// Change structure of the config to be transformed
-			let tmpConfig = {};
-			Object.keys(configModule).forEach(prop => {
-				const configs = Object.keys(configModule[prop].configuration);
-				configs.forEach(config => {
-					tmpConfig[config] = configModule[prop].configuration[config];
+			const tmpConfig: object = {};
```

Comment:
this breaks the build

Reply:
How? This is already present in master code.

https://github.com/webpack/webpack-cli/blob/ca504de8c7c0ea66278021b72fa6a953e3ffa43c/packages/utils/modify-config-helper.js#L73